language/compiler/ir-to-bytecode/syntax/src/lexer.rs

```diff
@@ -434,7 +432,7 @@ fn get_name_token(name: &str) -> Tok {
         "old" => Tok::Old,
         "public" => Tok::Public,
         "requires" => Tok::Requires,
-        "resource" => Tok::Resource,
+        "copy" => Tok::Copy,
```

Comment:
I don't know how "copyable" got added at the end, but this was supposed to be in alphabetic order to make it easier to find stuff. Can you move this to the right place?

Reply:
Ah probably a bad copy paste or something. I don't know. I'll fix it